### PR TITLE
Added zome integrity check

### PR DESCRIPTION
**Pull request by sphinxc0re**
- [ ] I have added a summary of my changes to the changelog

### CODE REVIEW

File: conductor_api/src/holochain.rs [hunk 1]
```diff
@@ -108,6 +112,21 @@ impl Holochain {
     /// create a new Holochain instance
     pub fn new(dna: Dna, context: Arc<Context>) -> HolochainResult<Self> {
         let instance = Instance::new(context.clone());
+
+        for zome in dna.zomes.values() {
```
Comment (zippy) [inline conductor_api/src/holochain.rs]: I think this needs to be broken out into it's own function for use by conductors, not as part of `new()` function.

This way makes it impossible to instantiate a instance unless the git hashes match.  I think this actually has to be configuration call, i.e. maybe there's a "strict" mode.  For now.  I think we should simply have our conductors spit out a bit warning if the hashes don't match.

Comment (sphinxc0re) [reply conductor_api/src/holochain.rs]: To get an understanding of what is needed here: How and where would such a function be called? Where should it go? Should I define it on `Zome`?

Comment (lucksus) [reply conductor_api/src/holochain.rs]: Yes. Or: instead of return an error below we just `println!` the version mismatch for now.

**Comment by sphinxc0re** [comment]
@lucksus sure, I can just `eprintln!` an error message, which prints it to `stderr`

**Pull request merged**